libtelemetry/src/main/java/com/mapbox/android/telemetry/SchedulerFlusherFactory.java

```diff
@@ -25,4 +29,19 @@ SchedulerFlusher supply() {
     return new AlarmSchedulerFlusher(context, alarmManager, alarmReceiver, requestCode);
     // }
   }
+
+  private void checkUpdatePeriod() {
+    try {
+      ApplicationInfo appInformation = context.getPackageManager().getApplicationInfo(context.getPackageName(),
+        PackageManager.GET_META_DATA);
+      if (appInformation != null && appInformation.metaData != null) {
+        boolean adjustWakeUp = appInformation.metaData.getBoolean(KEY_META_DATA_WAKE_UP, false);
+        if (adjustWakeUp) {
+          FLUSHING_PERIOD_IN_MILLIS = 600000;
```

Comment:
Could you split this value into a more readable format? Also, the name of the value shouldn't be in all caps since it's mutable, for readability sake as well.